content/en/blog/custom-metrics-autoscaling-new-deploy.md

```diff
@@ -0,0 +1,596 @@
++++
+title = "Autoscaling on Custom Metrics in New Deploy Functions"
+date = "2022-05-11T14:43:46+05:30"
+author = "Ankit Chawla"
+description = "Implementing custom metrics for HPA scaling"
```

Comment:
Let's change the description to "_Let us see how to enable autoscaling for serverless functions with custom metrics_"